README.md

```diff
@@ -42,19 +42,32 @@ on first run, this will be populated with a basic config if it doesn't exist.
 The default config is:
 
 ```toml
-# How often to check battery status, in seconds.
-interval_secs = 30
+# How often to check battery status, in milliseconds.
+interval = 30000
```

Comment:
Removing the unit is confusing. Instead, let's use `#[serde_as(as = "humantime_serde::Duration")]`.

Reply:
But it will be a breaking change anyway, right? Because we can't leave the _secs suffix then. The user will have to consult the docs anyway as mentioned in another reply so this is okay being left like this in my opinion. Maybe this can be done some time later other than in this PR.

Reply:
Milliseconds are pretty standard for specifying time. More like a default I feel really unless mentioned explicitly (like *_secs).

Reply:
I don't think many people are going to need values large enough for this to justify being able to set something like "2h" instead. A minutes at most make sense for this config.

Reply:
The changes needed would be:

- If interval_secs is present, populate interval wtih it and print a deprecation message
- If interval is present, parse it with humantime_serde::Duration or similar
- If both are present, exit

In any case, the units should always be present. Over the years while developing systemd we've had countless problems with directives like `*USec`, `*Sec` and the like. The best way to avoid that in future is to always require units.

Reply:
I see, I have to agree with you then on units being always present, with your level of experience.

Regarding breaking-change right away vs. deprecation message first, what will we do about the rest of the changes which might also be breaking?

Reply:
What if we simply call it `interval_ms` instead? Similarly for notifications.

Reply:
I'm okay with either not changing it, or using interval with humantime parsing, but let's not break backwards compat just for that when I don't think there's any demand for ms granularity.

Reply:
I imagine using floats for interval_secs is out of question? We can convert to ms/us while parsing the config. Otherwise I will try doing humantime with depraction notice, and if I fail we can let it be as it is. Thanks.